src/test/java/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParseTreeTest.java

```diff
@@ -78,6 +81,10 @@ private static String getDocPath(String filename) throws IOException {
         return getPath("javadocTags" + File.separator + filename);
     }
 
+    private static String getOutputDocPath(String filename) throws IOException {
+        return getDocPath("output" + File.separator + filename);
```

Comment:
I don't like `output` as files in this folder are not something we produce. I propose using `tree` or `trees` as a folder name and `<input file name>Tree` for files inside.